scripts/run_ginkgo.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -e
+
+# Load the versions
+SUBNET_EVM_PATH=$(
```

Comment:
Add comments that this assumes there is a local network running and how to provide environment variables to the ginkgo tests